codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/protocols/rpcv2cbor/RPCV2CBORHttpBindingResolver.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package software.amazon.smithy.aws.swift.codegen.protocols.rpcv2cbor
+
+import software.amazon.smithy.model.pattern.UriPattern
+import software.amazon.smithy.model.shapes.OperationShape
+import software.amazon.smithy.model.traits.HttpTrait
+import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
+import software.amazon.smithy.swift.codegen.integration.protocols.core.StaticHttpBindingResolver
+
+class RPCV2CBORHttpBindingResolver(
+    context: ProtocolGenerator.GenerationContext,
+    defaultContentType: String
+) : StaticHttpBindingResolver(context, rpcv2CborHttpTrait, defaultContentType) {
+
+    companion object {
+        private val rpcv2CborHttpTrait: HttpTrait = HttpTrait
+            .builder()
+            .code(200)
+            .method("POST")
+            .uri(UriPattern.parse("/"))
+            .build()
+    }
+
+    override fun httpTrait(operationShape: OperationShape): HttpTrait = HttpTrait
+        .builder()
+        .code(200)
+        .method("POST")
+        .uri(UriPattern.parse("/service/${getServiceName()}/operation/${operationShape.id.name}"))
```

Comment:
SEP requires particular URI pattern

Reply:
SEP includes `{prefix?}` portion before `/service`; it doesn't affect routing, but nevertheless can be configured by services for internal use / metrics. Might need to look into where that comes from in the model and add accordingly.

Reply:
It's not part of what we need to worry about constructing in the URI - it's things like if the endpoint is http://example.com/baz, the URL is http://example.com/baz/service/...

(Confirmed with Smithy team)